blog

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="styles.css">  
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  <title></title>
+</head>
+
+<body>
+
+<header>
+<div class="header-holder">
+<section class="left-side-header">
+<img src="images/adoptadog-logo.svg">
+<p>Seattle's premiere dog adoption agency</p>
+</section>
+<section class="right-side-header">
+  <p><i class="fas fa-shopping-cart"></i> $1234.56  </p>
```

Comment:
You'll want to use the asses we've provided. You can use SVG's just like an image tag. You should take out the font awesome out of your code, we're doing thins the hard way. =)